### PR TITLE
cleaning up old mutationMapper related files

### PR DESCRIPTION
**Pull request by pieterlukasse**
# What? Why?

Fix cBioPortal/mutation-mapper#51

Backend fix can be found at https://github.com/cBioPortal/cbioportal/pull/6314

### CODE REVIEW

**Comment by pieterlukasse** [comment]
@onursumer can you please review and test this one? Thanks!

**Comment by onursumer** [comment]
@pieterlukasse Network tab is currently an `iframe` version of the legacy app, I think for now it is okay to ignore the files under the `network/files` directory. Eventually, we will remove the whole directory and replace it with a proper react implementation.

File: src/shared/components/structureViewer/StructureVisualizer3D.spec.ts [hunk 1]
```diff
@@ -125,7 +125,7 @@ describe('StructureVisualizer3D', () => {
             "Background color should be set to 0xFFFFFF");
 
         assert.isTrue($3Dmol.download.calledWith("pdb:3PXE", viewer, sinon.match({pdbUri: props.pdbUri})),
-            "Structure download should be requested for pdb:3PXE from http://www.rcsb.org/pdb/files/");
+            "Structure download should be requested for pdb:3PXE from https://files.rcsb.org/view/");
```
Comment (onursumer) [inline src/shared/components/structureViewer/StructureVisualizer3D.spec.ts]: :+1:

**Comment by alisman** [comment]
@pieterlukasse @onursumer network tab iframe sources (frontend) are now entirely contained in frontend project.  it still uses legacy services though.

**Pull request merged**